packages/composite-checkout-wpcom/test/form-field-annotation.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { matchers } from 'jest-emotion';
+import { ThemeProvider } from 'emotion-theming';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldAnnotation from '../src/components/form-field-annotation';
+
+// Add the custom matchers provided by 'jest-emotion'
+expect.extend( matchers );
+
+const theme = {
+	colors: {
+		textColor: 'blue',
+		textColorLight: 'lightblue',
+		borderColor: 'black',
+		error: 'red',
+	},
+	weights: { bold: '15pt' },
+};
+
+describe( 'FormFieldAnnotation', () => {
+	describe( 'with no error and not disabled', () => {
+		let MyFormFieldAnnotation = null;
+
+		beforeEach( () => {
+			MyFormFieldAnnotation = () => (
+				<ThemeProvider theme={ theme }>
+					<FormFieldAnnotation
+						formFieldId={ 'fieldId' }
+						labelText={ 'A Label' }
+						labelId={ 'labelId' }
+						normalDescription={ 'A description' }
+						descriptionId={ 'descriptionId' }
+						errorDescription={ 'An Error Message' }
+						isError={ false }
+						isDisabled={ false }
+						className={ 'test__annotation_class' }
+					>
+						<span id={ 'fieldId' }>{ 'child contents' }</span>
```

Comment:
As mentioned above, you could do this without leaving JSX.

```suggestion
						<span id="fieldId">child contents</span>
```